### ISSUE TITLE
Wrong browser_version and engine_string detections for MS Edge

### ISSUE THREAD

**Issue by motxx** [oalders/http-browserdetect#179]
I think `browser_version` and `engine_string` are wrongly detected in Chromium-based MS Edge.

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36 Edg/80.0.361.111

# output
browser:edge, browser_version:5.0, engine_string:EdgeHTML, engine_version:80.0
```
- `browser_version` should be `80.0`.
- `engine_string` should be `WebKit` (not `Blink` as far as I read [the document](https://github.com/oalders/http-browserdetect#engine_string))

I used the followings as reference:
* https://en.wikipedia.org/wiki/Microsoft_Edge#Release_history
* https://docs.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel

**Comment by oalders** [comment]
Yes, that looks to be wrong. Thanks for reporting. I would accept a patch to fix this. :)

**Comment by motxx** [comment]
Thank you for your reply. I created the pull request here: https://github.com/oalders/http-browserdetect/pull/180

**Comment by oalders** [comment]
Closed via #180
